frontend/.eslintignore

```diff
@@ -1,5 +1,5 @@
 node_modules
 dist
 build
-lib
-__generated__
\ No newline at end of file
+core-api-client/lib
```

Comment:
@neb42 maybe another lib folder I forgot?

Reply:
That path is wrong, it should be `packages/core-api-client/lib`, but a better path would be `**/lib`

Reply:
Same for `dist` and `build` above and probably add an additional `**/node_modules` (keeping the current `node_modules`).

I'm not sure if `**/` will include the current directory

Reply:
**/lib would ignore all lib folders, but the code of the client package is also in a lib folder

Reply:
I'll see if I can change that, don't know why it's like that

Reply:
> **/lib would ignore all lib folders, but the code of the client package is also in a lib folder

ok then we should just explicitly add each lib folder